--- /Users/mpl/temp/3/p7zip_9.20.1/CPP/7zip/UI/Common/ArchiveOpenCallback.h	2009-05-31 10:10:03.000000000 +0200
+++ /Users/mpl/Documents/workspace/p7zip_9.20.1/CPP/7zip/UI/Common/ArchiveOpenCallback.h	2016-05-24 22:14:57.000000000 +0200
@@ -91,8 +91,8 @@
   void Init(const UString &folderPrefix, const UString &fileName)
   {
     _folderPrefix = folderPrefix;
-    if (!_fileInfo.Find(_folderPrefix + fileName))
-      throw 1;
+    //if (!_fileInfo.Find(_folderPrefix + fileName))
+    //  throw 1;
     FileNames.Clear();
     _subArchiveMode = false;
     TotalSize = 0;
