diff -u -r p7zip_9.20.1_org/CPP/7zip/UI/Common/ArchiveOpenCallback.h p7zip_9.20.1/CPP/7zip/UI/Common/ArchiveOpenCallback.h
--- p7zip_9.20.1_org/CPP/7zip/UI/Common/ArchiveOpenCallback.h	2009-05-31 10:10:03.000000000 +0200
+++ p7zip_9.20.1/CPP/7zip/UI/Common/ArchiveOpenCallback.h	2016-08-17 15:49:59.000000000 +0200
@@ -91,8 +91,8 @@
   void Init(const UString &folderPrefix, const UString &fileName)
   {
     _folderPrefix = folderPrefix;
-    if (!_fileInfo.Find(_folderPrefix + fileName))
-      throw 1;
+    //if (!_fileInfo.Find(_folderPrefix + fileName))
+    //  throw 1;
     FileNames.Clear();
     _subArchiveMode = false;
     TotalSize = 0;
diff -u -r p7zip_9.20.1_org/CPP/Windows/FileFind.cpp p7zip_9.20.1/CPP/Windows/FileFind.cpp
--- p7zip_9.20.1_org/CPP/Windows/FileFind.cpp	2011-01-11 21:55:10.000000000 +0100
+++ p7zip_9.20.1/CPP/Windows/FileFind.cpp	2016-08-18 12:01:40.000000000 +0200
@@ -272,8 +272,18 @@
     SetLastError(ERROR_PATH_NOT_FOUND);
     return false;
   }
- 
+  
   my_windows_split_path(nameWindowToUnix(wildcard),_directory,_pattern);
+    
+  //ML1
+  try
+  {
+      if (!fillin_CFileInfo(fileInfo,_directory,_pattern))
+      {
+          return true;
+      }
+  } catch (...) {}
+ 
   
   TRACEN((printf("CFindFile::FindFirst : %s (dirname=%s,pattern=%s)\n",wildcard,(const char *)_directory,(const char *)_pattern)))
 
